db/schema.rb

```diff
@@ -1294,11 +1297,13 @@
     t.datetime "updated_at",                      :null => false
     t.integer  "bundle_logger_id"
     t.integer  "console_logger_id"
+    t.string   "key"
```

Comment:
I don't have this "key" in my schema.rb. I suspect that your database had some migrations run on it that didn't make it into master. The new fields on interactives are legitimate, but the others changes here seem to be old. Perhaps just take this schema.rb change out of this PR. Then configure a new blank db, run the migrations and make a new PR with the missing schema bits.